cardano-api/src/Cardano/Api/IPC/NtcVersionOf.hs

```diff
@@ -0,0 +1,21 @@
+module Cardano.Api.IPC.NtcVersionOf
+  ( NtcVersionOf (..)
+  , MinNodeToClientVersion
+
+  -- *** Error types
+  , UnsupportedNtcVersionError(..)
+  ) where
+
+import           Data.Eq (Eq)
+import           Text.Show (Show)
+import           Ouroboros.Network.NodeToClient.Version (NodeToClientVersion (..))
+
+-- | The query 'a' is a versioned query, which means it requires the Node to support a minimum
```

Comment:
Let's expand this comment to include more about the NodeToClient versioning i.e why do we have it? We can probably also point to documentation in the networking code that explains how the handshaking etc works.

Reply:
More documentation added.